.drone.star

```diff
@@ -741,17 +198,17 @@ def main(ctx):
     return pipelines + deploys + checkStarlark()
 
 def beforePipelines(ctx):
-    return yarnlint() + changelog(ctx) + website(ctx) + cacheOcisPipeline(ctx)
+    return []
 
 def stagePipelines(ctx):
     acceptancePipelines = acceptance(ctx)
     if acceptancePipelines == False:
         return unitTests(ctx)
 
-    return unitTests(ctx) + acceptancePipelines
```

Comment:
How can this PR fulfill its goal if the unitTest aren't run anymore? Plus, once again, make sure to kill the pipeline if the pipelines don't stop after the unit tests @kiranparajuli589

Reply:
just to fix webui suites :) i will revert these changes